1.9/overview/architecture/boot-sequence.md

```diff
@@ -0,0 +1,94 @@
+---
+post_title: Boot Sequence
+nav_title: Boot Sequence
+menu_order: 6
+---
+
+During installation, the DC/OS components are all started in parallel but initialize and become responsive in a particular sequence because of interdependencies.
+
+3DT monitors component health on each node. Once all node components are healthy, 3DT reports that that node is healthy. Once all master nodes are healthy, 3DT reports that the cluster is healthy.
```

Comment:
3DT should be written as `Diagnostics` or `Diagnostics Service`. 3DT is the internal component name.